www2/server/api/pricing/pricing.spec.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+var should = require('should');
```

Comment:
you should use const everywhere, not var

Reply:
Done